runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/SkillChallengeClue.java

```diff
@@ -150,7 +150,7 @@
 		new SkillChallengeClue("Smith a runite med helm.", item(ItemID.HAMMER), item(ItemID.RUNITE_BAR)),
 		new SkillChallengeClue("Teleport to a spirit tree you planted yourself."),
 		new SkillChallengeClue("Create a Barrows teleport tablet.", item(ItemID.DARK_ESSENCE_BLOCK), xOfItem(ItemID.BLOOD_RUNE, 1), xOfItem(ItemID.LAW_RUNE, 2), xOfItem(ItemID.SOUL_RUNE, 2)),
-		new SkillChallengeClue("Kill a Nechryael.", "slay a nechryael"),
+		new SkillChallengeClue("Kill a Nechryael.", "slay a nechryael in the slayer tower"),
```

Comment:
This is missing the trailing period.
```suggestion
		new SkillChallengeClue("Kill a Nechryael.", "slay a nechryael in the slayer tower."),
```